### PR TITLE
show message for emtpy dataset search [FIX #1044]

### LINKED ISSUE #1044: A search with unmatched results return a "datasets 1 to 0 of 0" label
## Detailed Description

As seen on https://www.data.gouv.fr/fr/topics/agriculture-et-alimentation/datasets?license=fr-lo&granularity=poi

image

## Context

When a theme search returns no results, I want to see a message stating there is no resealt, so I can try something else instead .

## Possible Implementation

N/C

## Your Environment

* Browser Name and version: N/A
* Operating System and version (desktop or mobile): N/A

/via @gregoirenovel

### PR DESCRIPTION
**Pull request by taniki**
image

### CODE REVIEW

File: udata/templates/dataset/list.html [hunk 1]
```diff
@@ -15,12 +15,16 @@
 
 {% block breadcrumb %}
     <li class="active">
-        {{ _('Datasets') }}
+			{% if datasets.total == 0 %}
```
Comment (noirbizarre) [inline udata/templates/dataset/list.html]: Even if there is no results, the breadcrum should be `Datasets`.

File: udata/templates/dataset/list.html [hunk 2]
```diff
@@ -15,12 +15,16 @@
...
+				<small>{{ _('No dataset') }}</small>
+			{% else %}
+				{{ _('Datasets') }}
         <small>{{ _('%(start)s to %(end)s on %(total)s',
```
Comment (noirbizarre) [inline udata/templates/dataset/list.html]: But the paging details can be hidden instead

File: udata/templates/dataset/list.html [hunk 3]
```diff
@@ -43,7 +47,15 @@
 
 {% block main_content %}
 <div class="row">
-    <div class="col-md-8 col-lg-9 smaller">
+
+		{% if not datasets %}
```
Comment (noirbizarre) [inline udata/templates/dataset/list.html]: Express the opposite, it should be clearer: `if datasets` ... `else` ... (positive condition and nominal case first)

**Comment by taniki** [comment]
image

squashed and ready. it should also fix the messed up indenting

**Comment by taniki** [comment]
related do #1044 

@noirbizarre 🆙

**Pull request merged**